src/test/java/com/exasol/adapter/dialects/rls/RowLevelSecurityDialectIT.java

```diff
@@ -6,10 +6,15 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 import java.nio.file.Path;
-import java.sql.*;
+import java.sql.Connection;
```

Comment:
Please correct this manually and I will check why the formatter settings are off again.

Reply:
done